src/client/client.ts

```diff
@@ -1,7 +1,7 @@
 // This file runs in the browser.
 
 // injected by serverPluginHmr when served
-declare const __PORT__: number
+declare const __HOST__: string
```

Comment:
We should keep the `__PORT__` and let `__HOST__` fallback to the original behavior.

Reply:
@antfu the original behavior was introduced in https://github.com/underfin/vite/commit/bb18db2b890ea7e95d57b0ce1c623001140f4eac

The new behavior is another way to do samething, but instead of get hostname and port from browser, it will get from directly Vite server.

In old behavior `${socketProtocol}://${location.hostname}:${__PORT__}`, `location` have context from browser.

In this new behavior `${socketProtocol}://${ctx.host.toString()}`, `ctx` will have context from Vite server.

I think that we can revert https://github.com/underfin/vite/commit/bb18db2b890ea7e95d57b0ce1c623001140f4eac and just let this behavior.

What do you thing?